isbn/malibu_light.css

```diff
@@ -0,0 +1,381 @@
+/* FARBEN
+
+Dunkelgrau #222429
+Mittelgrau #34373e
+Hellgrau #44454c
+Hell #F1EDEB
+
+*/
+
+
+
+
+
+/***** ALLGEMEINE EINSTELLUNGEN *****/
+
+body {
+
+  margin: 0;
+
+  font-family:  Arial, Verdana, sans-serif;
+
+  background-color: #fff;
+
+}
+
+
+
+/* RESET */
+
+* {
+
+margin: 0;
```

Comment:
No indentation here?

Reply:
You are right, indentation is missing here. I think also that we could delete some empty lines. Do you have a script for that or want to do it manually? Feel free to do so...